pkg/plugins/connection/ssh_test.go

```diff
@@ -0,0 +1,39 @@
+package connection
+
+import (
+	"context"
+	"testing"
+
+	"github.com/nornir-automation/gornir/pkg/gornir"
+	"github.com/nornir-automation/gornir/pkg/plugins/logger"
+	"golang.org/x/crypto/ssh"
+)
+
+func TestTime(t *testing.T) {
```

Comment:
haha maybe it's better to call it `TestSSHOpen` ;)

Reply:
Whoops, I might have copy-pasta'd the boilerplate.  :)